dev-requirements.txt

```diff
@@ -1,7 +1,7 @@
 # install latest changes in dbt-core
 # TODO: how to automate switching from develop to version branches?
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-core&subdirectory=core
-git+https://github.com/dbt-labs/dbt-core.git#egg=dbt-tests-adapter&subdirectory=tests/adapter
+git+https://github.com/dbt-labs/dbt-core.git@4122f6c308c88be4a24c1ea490802239a4c1abb8#egg=dbt-core&subdirectory=core
```

Comment:
Revert the version of `dbt-core` to immeidately prior to this PR: https://github.com/dbt-labs/dbt-core/pull/8856.